README.md

```diff
@@ -1 +1,181 @@
-# Javascript-Capstone_AlbertandJuw
\ No newline at end of file
+# Javascript-Capstone_AlbertandJuw
+<a name="readme-top"></a>
+<div align="center">
+<img src="kanbanboard-logo.png" alt="kanbanboard logo" width="140"  height="auto" />
+
+  <h2>Kanban Board Project App</h2>
+  <br/>
+
+  <h3><b>Kanban Board Project App</b></h3>
```

Comment:
- Optional
I would strongly recommend giving a proper title to your project. Having a good README is super important as it is the first thing a recruiter will see. A title like **Meals DB** would be suitable for a project like this.